share/OpenMS/SCHEMAS/IdXML_1_4.xsd

```diff
@@ -249,7 +249,7 @@
 				<xs:documentation>significance threshold as calculated by the search engine</xs:documentation>
 			</xs:annotation>
 		</xs:attribute>
-		<xs:attribute name="spectrum_reference" type="xs:unsignedInt">
+                <xs:attribute name="spectrum_reference" type="xs:string">

```

Comment:
e.g. the documentation needs to be adapted to. also I don't know if we would need to increase the version number of idXML